mnefun/_report.py

```diff
@@ -62,23 +62,30 @@ def gen_html_report(p, subjects, structurals, run_indices=None,
                                     run_indices[si])
         has_pca = all(op.isfile(fname) for fname in pca_fnames)
 
+        # whitening and source localization
+        cov_name = op.join(p.work_dir, subj, p.cov_dir,
+                           safe_inserter(cov, subj))
```

Comment:
_TypeError: argument of type 'bool' is not iterable_
 >  File "..._report.py", line 67, in gen_html_report
>     safe_inserter(cov, subj))
>   File "..._paths.py", line 29, in safe_inserter
>     if '%s' in string:

Reply:
Should be fixed in latest commit but I can't test locally right now (until Monday)

Reply:
Thanks.